ERCS/erc-7399.md

```diff
@@ -4,7 +4,7 @@ title: ⚡ Flash Loans ⚡
 description: Interfaces and processes for flash loans
```

Comment:
I can't comment on the line above, but the title here is probably too similar to https://eips.ethereum.org/EIPS/eip-3156. You should differentiate this proposal from that one.